src/CodeStyle/VisualBasic/CodeFixes/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj

```diff
@@ -55,6 +55,9 @@
     <ProjectReference Include="..\..\Core\CodeFixes\Microsoft.CodeAnalysis.CodeStyle.Fixes.csproj" />
     <ProjectReference Include="..\Analyzers\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.vbproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Import Include="System.Threading.Tasks" />
```

Comment:
😕 Why did this change?

Reply:
The default set of imports has been changing between SDKs. I added a set during the move to the Preview 6 SDK and these are now needed moving to the Preview 7 SDK.

Reply:
Which hopefully will be addressed by https://github.com/dotnet/sdk/issues/19521